includes/view/ShiftEntry_view.php

```diff
@@ -224,8 +224,8 @@ function ShiftEntry_edit_view(
                 config('max_freeloadable_shifts')]);
         }
         $freeload_form = [
-            form_checkbox('freeloaded', __('Freeloaded') . ' <span class="bi bi-info-circle-fill text-info" data-bs-toggle="tooltip" title="' .
-                $freeload_info . '"></span>', $freeloaded),
+            form_checkbox('freeloaded', (!$freeloaded_by ? __('Freeloaded') : __('Freeloaded by %s', [User_Nick_render($freeloaded_by)])) . ' <span class="bi bi-info-circle-fill text-info" data-bs-toggle="tooltip" title="' .
```

Comment:
this checkbox is somehow not set when another user wants to edit the freeloaded flag; also when removing the comment & unchecking the box i as another user I get another `Attempt to read property "displayName" on null` error

Reply:
because you told me to change it to no instead of not null please figure out why this is broken yourself

Reply:
GRRRRRRR its disgusting